modules/custom/inope_policy_review/inope_policy_review.module

```diff
@@ -566,199 +542,6 @@ function inope_policy_review_update_query(&$form_state) {
 }
 
 /**
- * Retrieve admin table from cache if possible.
```

Comment:
Caching ran through my mind briefly but I knew views had a caching option that we could reimplement.